ci-operator/step-registry/ipi/conf/azure/ipi-conf-azure-commands.sh

```diff
@@ -60,3 +81,26 @@ EOF
     echo "${OUTBOUND_TYPE} is not supported yet" || exit 1
   fi
 fi
+
+version=$(getVersion)
```

Comment:
If you want to use `getVersion` to receive version value, `getVersion` should not print out any other things, only version value is required.